README.md

```diff
@@ -669,28 +669,29 @@ Input:
 
 Output: a json with the following field:
 - can_create: boolean. Whether user can create new alias
-- suffixes: list of string. List of alias `suffix` that user can use. If user doesn't have custom domain, this list has a single element which is the alias default domain (simplelogin.co).
+- suffixes: list of `[suffix, signed-suffix]`. List of alias `suffix` that user can use. The `signed-suffix` is necessary to avoid request tampering.
```

Comment:
@ntnhon this new endpoint adds the concept of `signed suffix` to avoid suffix tampering.